installers/certify.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+RELEASE_VERSION=${CI_BRANCH}
+
+with_client=false
+if [ $# -eq 1 ] && [ $1 = "--with_client" ] ; then
+    if [  !  -f /artifacts/build/dc/current-version.txt ] ; then
+        echo "Desktop client does not exist"
+        exit 1
+    fi
+    with_client=true
+fi
+
+if [ $with_client = true ]; then
```

Comment:
Can you move this block into the previous block, and remove references to `with_client`?